Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -330,6 +329,7 @@ theorem eLpNorm'_const (c : ε) (hq_pos : 0 < q) :
   suffices hq_cancel : q * (1 / q) = 1 by rw [hq_cancel, ENNReal.rpow_one]
   rw [one_div, mul_inv_cancel₀ (ne_of_lt hq_pos).symm]
 
+-- Generalising this to ENormedAddMonoid means requiring c ≠ ⊤.
```

Comment:
Does it? Statement looks right for `c = ⊤`, but I can imagine that in the proof you might have to do a case-split.

Reply:
Right: since I have performed this generalisation in #22721 already, let me just change the comment.
```suggestion
-- Generalising this to ENormedAddMonoid requires a case analysis whether ‖c‖ₑ = ⊤,
-- and will happen in a future PR.